test/sdkclient.go

```diff
@@ -80,6 +81,14 @@ func WithRootCA(certificate, key []byte) Option {
 	})
 }
 
+// WithID creates Option that overrides the validFrom timestamp used by the SDK client
+// when generating a certificate use by the client.
+func WithValidFrom(validFrom string) Option {
```

Comment:
I want to replace in future `ValidFrom` and `ValidFor` by `notBefore` and `notAfter` as these fields are defined in certificate specification.

Reply:
Added a TODO